ui/app/src/components/ToolsPanelEmbeddedAppViewButton/ToolsPanelEmbeddedAppViewButton.tsx

```diff
@@ -40,6 +40,7 @@ export function ToolsPanelEmbeddedAppViewButton(props: ToolsPanelEmbeddedAppView
   } = useEnhancedDialogState();
   const title = usePossibleTranslation(props.title);
   const widgetDialogPendingChangesCloseRequest = useWithPendingChangesCloseRequest(onClose);
+  const isWidgetEmbedded = props.widget?.configuration?.embedded;
```

Comment:
This component has no business reading the internals of widgets and their config.